plugins/iframe-resizer/host.js

```diff
@@ -16,7 +16,7 @@ export function hostResizer(host) {
 		};
 
 		const resizerScript = document.createElement('script');
-		resizerScript.src = 'https://s.brightspace.com/lib/iframe-resizer/3.6.5/iframeResizer.min.js';
+		resizerScript.src = 'https://s.brightspace.com/lib/iframe-resizer/3.6.6/iframeResizer.min.deumdified.js';
```

Comment:
You know, it occurs to me that if we didn't need to deumdify this, we could do something like ```import('../../node_modules/iframe-resizer/js/iframeResizer.min.js).then(resize)```
and just rely on its side effects to avoid the CDN business altogether. Oh well.